prometheus/pluginsrc/com/defold/extension/pipeline/Prometheus.java

```diff
@@ -106,12 +105,7 @@ private void unpackPrometheusSource() throws IOException {
 	}
 
 	@Override
-	public String create(String filePath, String input, String buildVariant) throws Exception {
-		return input;
-	}
```

Comment:
Now that we have two separate interfaces we don't need to provide an implementation for this.